web-api-shared/src/net.rs

```diff
@@ -67,6 +71,15 @@ impl<F> ExponentialJitterRetryPolicy<F> {
         }
     }
 
+    pub fn retry<MakeFut, Fut>(self, futures: MakeFut) -> RetryFuture<Self, MakeFut, Fut>
+    where
+        Self: RetryPolicy<Fut::Output>,
+        MakeFut: FnMut() -> Fut,
+        Fut: Future,
+    {
+        retry(self, futures)
+    }
+
```

Comment:
At some point I thought I needed to use a additional retry policy and added this method to allow more convenient usage. As it's still nice I kept that change.